hypothesis-python/docs/usage.rst

```diff
@@ -8,8 +8,11 @@ The only inclusion criterion right now is that if it's a Python library
 then it should be available on PyPI.
 
 You can find hundreds more from `the Hypothesis page at libraries.io
-<https://libraries.io/pypi/hypothesis>`_, and over a thousand
+<https://libraries.io/pypi/hypothesis>`_, and over a two-and-a-half thousand
```

Comment:
If you're going to be this specific, then style-wise it probably makes more sense to write out 2500 in numerals.